modules/text2colbert-jinaai/clients/jinaai.go

```diff
@@ -0,0 +1,84 @@
+//                           _       _
```

Comment:
Can you also add this to the default modules?

Reply:
what to add? this whole Jina vectorizer?

Reply:
text2colbert-jina should be enabled by default - there is a list of modules that are enabled by default somewhere :)

Reply:
not yet, this module can be enabled only when all other changes are done, so we have to wait with enabling it